operations/CSM_product_management/Configure_Non-Compute_Nodes_with_CFS.md

```diff
@@ -3,17 +3,35 @@
 Non-compute node (NCN) personalization applies post-boot configuration to the
 HPE Cray EX management nodes. Several HPE Cray EX product environments outside
 of CSM require NCN personalization to function. Consult the manual for each
-product to configure them on NCNs by referring to the [1.5 HPE Cray EX System
-Software Getting Started Guide S-8000](https://www.hpe.com/support/ex-gsg) on
-the HPE Customer Support Center.
+product to configure them on NCNs by referring to the
+[`1.5 HPE Cray EX System Software Getting Started Guide S-8000`](https://www.hpe.com/support/ex-gsg)
```

Comment:
Does the 1.5 HPE Cray EX System Software Getting Started Guide apply to 1.2?  Is there a CSM 1.x version that should be references instead?